DependencyInjection/Configuration.php

```diff
@@ -35,6 +35,7 @@ public function getConfigTreeBuilder()
                         ->scalarNode('recipient')->isRequired()->cannotBeEmpty()->end()
                         ->scalarNode('sender')->isRequired()->cannotBeEmpty()->end()
                         ->scalarNode('subject')->isRequired()->cannotBeEmpty()->end()
+                        ->booleanNode('send_on_warning')->defaultTrue()->end()
```

Comment:
wondering if this shouldn't be an array .. like maybe someone also does not want emails for unknown?